src/main/java/uk/org/tombolo/importer/ons/CensusImporter.java

```diff
@@ -90,6 +91,22 @@ protected String getDataUrl(String datasourceIdString, String geography) {
                 + "&" + "geography=" + GEOGRAPHIES.get(geography);
     }
 
+    @Override
+    public void importDatasource(@Nonnull String datasourceId, List<String> geographyScope, List<String> temporalScope,
+                                 List<String> datasourceLocation, @Nonnull List<SubjectRecipe> subjectRecipes, Boolean force) throws Exception {
+        OaImporter oaImporter = new OaImporter(config);
+        oaImporter.setDownloadUtils(downloadUtils);
+        for (SubjectRecipe subjectRecipe : subjectRecipes) {
+            if (!DatabaseJournal.journalHasEntry(JournalEntryUtils.getJournalEntryForDatasourceId(
+                "uk.org.tombolo.importer.ons.OaImporter", subjectRecipe.getSubjectType(), geographyScope,
+                    temporalScope, datasourceLocation))) {
+                oaImporter.importDatasource(subjectRecipe.getSubjectType(), Collections.EMPTY_LIST, Collections.EMPTY_LIST,
```

Comment:
The usage of the subject type as datasourceID here is straightforward but a bit misleading.